projects/packages/my-jetpack/_inc/hooks/use-notification-watcher/use-backup-needs-attention-notice.tsx

```diff
@@ -5,6 +5,7 @@ import { useContext, useEffect, useCallback } from 'react';
 import { NOTICE_PRIORITY_HIGH } from '../../context/constants';
 import { NoticeContext } from '../../context/notices/noticeContext';
 import { applyTimezone } from '../../utils/apply-timezone';
+import createCookie from '../../utils/create-cookie';
```

Comment:
Ahh, TIL about `createCookie`! Nice! Thanks @CodeyGuyDylan!

Reply:
Oh nevermind, I see you created it! (createCookie)
Still that's a nice improvement! 👍🙂